src/main/java/org/folio/circulation/domain/representations/ItemNotLoanableBlock.java

```diff
@@ -0,0 +1,26 @@
+
+package org.folio.circulation.domain.representations;
+
+import static org.folio.circulation.support.json.JsonPropertyFetcher.getDateTimeProperty;
+
+import org.joda.time.DateTime;
+
+import io.vertx.core.json.JsonObject;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public class ItemNotLoanableBlock {
+
+  private DateTime dueDate;
+
+  public static ItemNotLoanableBlock from(JsonObject representation) {
+
+    if (representation != null) {
+      return new ItemNotLoanableBlock(
+        getDateTimeProperty(representation, "dueDate"));
```

Comment:
Make sure that null value for dueDate (and absence of this filed) is handled correctly